[feat#2-Scheduler] 스프링 스케쥴러를 이용해 Github Commit 정보 저장

Ggyumalang opened the pull request:
## 작업 내용 (Content)
- 변경 사항 
  스프링 스케줄러를 이용해서 매일 00시 05분에 전일 날짜 - 현재 일자 00시 05분까지의 커밋 정보를 업데이트한다.

## Merge 전 필요 작업 (Checklist before merge)

## 희망 리뷰 완료 일 (Expected due date)
`2023. 04. 05. Wed`

## 팀원분들께
기존에 Jenkins 를 이용해 배치 기능을 구현하려했지만 시간이 촉박할 것 같아 우선 스프링 스케줄러로 구현하였으며 추후 Jenkins를 공부하여 구현해보도록 노력하겠습니다.
또한 스케쥴링 시간을 현재는 해당 클래스에 직접 입력해두었는데 협의 후 application-scheduler.yml 같은 설정 파일을 올려서 관리할 예정입니다.
또한, GithubAPI Test 부분이 깃헙 실제 정보가 들어가야해서 우선 주석처리하였습니다.

Ggyumalang committed:
feat : [feat#2-Scheduler] 스프링 스케쥴러를 이용해 Github Commit 정보 저장

스프링 스케줄러를 이용해서 매일 00시 05분에 전일 날짜의 커밋 정보를 업데이트한다.

xoals25 commented on user-api/src/main/java/zerobase/bud/github/scheduler/GetCommitInfoScheduler.java:
```diff
@@ -0,0 +1,47 @@
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class GetCommitInfoScheduler {
```
코드 정리하는 방식이 깔끔하고 좋은것 같아요!!
많이 배웁니다!
고생 많으셨어요!

Ggyumalang committed:
fix : [feat#2-Test관련 수정]

실제 깃헙 정보(깃헙 유저명, 깃헙 액세스토큰, 깃헙 이메일)이 들어가야 테스트 통과하오나 그 정보들을 외부에 보여줄 수 없기에 우선 주석처리를 함.

Sfer7 reviewed (approved):
아이고 확인이 늦었네요 ㅠㅠ
고생하셨습니다!
